doc/user_guide/user_guide.md

```diff
@@ -21,48 +21,52 @@ While this might sound counter-intuitive at first, all attributes of the error o
+If you should forget all attributes, `ExaError` will default to an "undefined error", telling the user to contact the software maker for a solution.
 
-Consequently `exaerror` does not restrict the attributes at runtime but rather tries to make the best possible error message out of what it is given.
+Consequently `ExaError` does not restrict the attributes at runtime but rather tries to make the best possible error message out of what it is given.
 
 ## Error Object Construction Variants
 
 There are different variants for creating an error object. Which one you prefer is a matter of taste mainly because the yield the same result.
 
-### Lua-style Object Initialization
+### Constructor `new`
 
-An established way in Lua to initialize objects is to provide a table with the `new` call that will then be turned into the object.
+The constructor method `new` has the following signature:
 
 ```lua
-local errorobj = exaerror:new({
-    code = "E-IO-13",
-    message = "Need {{needed}} MiB space, but only {{remaining}} MiB left on device {{device}}.",
-    parameters = {needed = {value = 500.2}, remaining = {value = 14.8}, device = {value = "/dev/sda4"}},
-    mitigations = {"Delete some unused files.", "Move to another device."}
-})
+ExaError:new(code, message, parameters, mitigations)
 ```
 
-As you can see in the example above, you can simply pass the error attributes as table when calling `exaerror:new`.
-
-The predefined table keys are
+With the following parameters:
 
 * `code`: machine-readable unique code (string)
 * `message`: Error description either as static string or containing placeholders in double curly brackets.
 * `parameters`: table of parameter values be used to replace placeholders in the message and mitigations
 * `mitigations`: list of hints on how to fix the error (array of strings), optionally containing placeholders
 
-The `parameters` must have a field `value`. Optionally you can explain what a parameters means using the field `description`.
+Here is an example that uses all constructor parameters.
+
+```lua
+local errorobj = ExaError:new(
+        "E-IO-13",
+        "Need {{needed}} MiB space, but only {{remaining}} MiB left on device {{device}}.",
+        {needed = {value = 500.2}, remaining = {value = 14.8}, device = {value = "/dev/sda4"}},
+        {"Delete some unused files.", "Move to another device."}
+)
+```
+
+The `parameters` must have a field `value`. Optionally you can explain what a parameter means using the field `description`.
 
 ### Error Object Builder
 
-Alternatively you can use a builder provided with `exaerror` to construct an error object.
+Alternatively you can use builder-style construction provided with `ExaError` to create an error object.
 
 Strictly speaking it is a combination between the builder pattern and regular setters since Lua 5.1 does not have immutable tables.
```

Comment:
Do we still support Lua 5.1?

Reply:
For this base module? Yes. It is useful outside of the Exasol context too.